packages/interaction/src/interactiveTarget.ts

```diff
@@ -92,42 +92,8 @@ export interface InteractiveTarget {
  *      );
  */
 export const interactiveTarget: InteractiveTarget = {
-
```

Comment:
Can you revert all these changes to interactiveTarget? Events should conflict with it.

Reply:
I don't see the purpose of duplicating these properties in the documentation? They have the same signature and description?